charts/merlin/values.yaml

```diff
@@ -38,12 +38,18 @@ feastCoreApi:
 
 imageBuilder:
   clusterName: "test"
-  buildContextURI: "git://github.com/gojek/merlin.git#refs/tags/v0.1"
-  baseImage: ""
-  dockerfilePath: "./Dockerfile"
-  predictionJobBuildContextURI: "git://github.com/gojek/merlin.git#refs/tags/v0.1"
-  predictionJobBaseImage: "gojek/mlp/merlin-pyspark:v0.4.1"
-  predictionJobDockerfilePath: "docker/app.Dockerfile"
+  baseImages: 
```

Comment:
there would be another values file under `charts/merlin/ci`. You would need to modify that too.